scripts/parser.js

```diff
@@ -0,0 +1,44 @@
+const jobsModel = require('../app/models/jobsModel.js');
+const MBO = require('mindbody-sdk');
+ 
+const mbo = new MBO({
+    ApiKey: process.env.API_KEY, // from portal
+    SiteId: parseInt(process.env.SITE_ID) //thisis the sandbox account
+});
+
+/**
+ * Iterates through provided class information from Mindbody API
+ * and updates or inserts jobs to the database
+ */
+async function processResponse(err,data){
+    if (err) {
+        console.error("Failed to retrieve information about classes", err);
+    } else {
+        const classes = data.Classes
+        await classes.map(async function(class_json) {
+            const scheduledJobs = await jobsModel.getJobsById(class_json.Id)
+            const numberOfJobs = scheduledJobs.length
```

Comment:
it's not necessary to have a variable to keep track of the length of the array, but since you already did it, we can use this to make the if statement more readable:
```
const isJobExist = scheduledJobs.length === 0;
```